CONTRIBUTING.md

```diff
@@ -0,0 +1,105 @@
+<img width="94" height="71" src="logo.svg" title="Autoprefixer logo by Anton Lovchikov">
+
+# Contributing to Autoprefixer
+Please feel free to contribute to Autoprefixer by submitting Pull Requests to add new features, ehancements, bug fixes, etc. If you're not sure whether a PR should be created for something, [open an issue](https://github.com/postcssAautoprefixer/issues) to ask about it first. Check the specific section depending on the type of contribution you want to make.
+
+ * [Getting started](#getting-started)
+ * [Adding a new prefix](#adding-a-prefix)
+ * [Changing an existing prefix](#changing-an-existing-prefix)
+ * [Fixing bugs](#fixing-bugs)
+ * [Filing issues](#filing-issues)
+ * [Running Tests](#running-tests)
+ * [Creating a Hack](#creating-a-hack)
+ * [Tips and guidelines](#guidelines-and-tips)
+
+Once your PR is accepted it will incorporated into the next main release of Autoprefixer.
```

Comment:
Is it important? Maybe we can remove this block?

Reply:
I don't think it's particularly important, nevertheless I find that contributors usually wants to know when their contribution will be incorporated into the product. For example, If I submitted a PR to add support for a new CSS feature, I would like to know when will it be available to see that the feature is now fully supported.